pypiserver/Chart.yaml

```
@@ -1,7 +1,8 @@
+apiVersion: v1
 name: pypiserver
 home: https://github.com/pypiserver/pypiserver
-version: 1.1.0
-appVersion: 1.3.0
+version: 1.2.0
```

podLabels replaces deployment.labels, it's a breaking change so I'd be more confortable bumping a major here

```suggestion
version: 2.0.0
```